sdks/python/container/run_validatescontainer_py3.sh

```diff
@@ -82,7 +83,7 @@ SDK_LOCATION=$(find dist/apache-beam-*.tar.gz)
 # Run ValidatesRunner tests on Google Cloud Dataflow service
 echo ">>> RUNNING DATAFLOW RUNNER VALIDATESCONTAINER TEST"
 python setup.py nosetests \
-  --attr Py3IT \
+  --attr ValidatesContainer \
```

Comment:
Ok, I totally missed this change in the earlier revision, now it is clear what keeps WordCount running, thanks.